x-pack/plugins/secops/public/components/timeline/timeline.test.tsx

```diff
@@ -22,62 +24,48 @@ describe('Timeline', () => {
     sortDirection: 'descending',
   };
 
+  const mocks = [
+    {
+      request: { query: eventsQuery },
+      result: {
+        data: {
+          evenst: mockECSData,
```

Comment:
This is spelled wrong. Surprised this mock is working. It might be turned off. `evenst -> events`

Reply:
oh you are right, I am wondering why

Reply:
It is running!!!

![image](https://user-images.githubusercontent.com/189600/48240874-83c91280-e3a2-11e8-81f3-59e695dae5df.png)